charts/connect/crds/onepassworditem-crd.yaml

```diff
@@ -2,8 +2,6 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: onepassworditems.onepassword.com
-  labels:
-    app.kubernetes.io/name: onepassword-connect
```

Comment:
I'm not sure why we had this here, so I cannot confirm whether its deletion is okay. It seems like we never had this on the CRD defined in the operator repo (https://github.com/1Password/onepassword-operator/blob/main/deploy/crds/onepassword.com_onepassworditems_crd.yaml).

@jillianwilson do you have more context on this?

@tomjohnburton, was the reason you introduced this change that it isn't in the operator either?

Reply:
This is safe to be removed